client_options.go

```diff
@@ -22,7 +22,8 @@ type ClientOption func(*ClientOptions)
 
 // ClientOptions encapsulate the configurable parameters on a Twirp client.
 type ClientOptions struct {
-	Hooks *ClientHooks
+	Hooks          *ClientHooks
+	SkipPathPrefix bool // if true, skip the default "/twirp" path prefix
```

Comment:
nit: I would name this `NoPathPrefix`

Reply:
I rested on this idea for a few days (allow baseURL to have any prefix, and allow to remove the "/twirp" prefix)... but I am not happy with it. Allowing the server to handle any route with any suffix is not entirely backwards compatible (sending requests like "/twirp/foo/bar/arbitrary/prefix/pck.Svc/Mtd" will also work, which may not be desired).

I will change the implementation to keep the allowed baseURL to be scheme and domain only (e.g. "http://example.com") and the prefix is provided through a client and service option, that is "/twirp" by default, so that servers will work exactly like they do now (only allow requests on the exact path) but still allow for other prefixes, and they need to be explicitly specified.